editor/src/main/java/net/rpgtoolkit/editor/editors/TileEditor.java

```diff
@@ -1,83 +0,0 @@
-/**
- * Copyright (c) 2015, rpgtoolkit.net <help@rpgtoolkit.net>
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-package net.rpgtoolkit.editor.editors;
-
-import java.awt.*;
-import java.io.File;
-import javax.swing.*;
-import javax.swing.event.ChangeEvent;
-import javax.swing.event.ChangeListener;
-
-import net.rpgtoolkit.common.assets.Tile;
-import net.rpgtoolkit.common.assets.TileSet;
-
-public class TileEditor extends JInternalFrame

```

Comment:
Looks like this wasn't anywhere near finished, so I don't feel bad about losing it. If we can import the most widespread formats, we can always tell people to use Paint.NET, GraphicsGale, PS, or whatever they like that is actually designed for image editing and can export pngs and the like.